content/chapter-destinations/clickhouse/_index.md

```diff
@@ -157,6 +157,51 @@ You can find the available column types in the [official ClickHouse documentatio
 
 {{% include-headless "chunk/option-destination-send-timezone.md" %}}
 
+## server-side-schema()
+
+|          |                            |
+| -------- | -------------------------- |
+| Type:    | string |
+| Default: | - |
+
+By default, sending data to ClickHouse doesn't propagate the type information of the data fields. The `server-side-schema()` option provides a solution for that using the [ClickHouse format schema](https://clickhouse.com/docs/interfaces/formats#formatschema).
+
+1. Create a `.proto` file that matches the `schema()` of your {{< product >}} configuration. For details on formatting, see the [ClickHouse documentation](https://clickhouse.com/docs/interfaces/formats/Protobuf#example-usage). For example:
+
+    ```json
+    syntax = "proto3";
+
+    message MessageType {
+      uint32 user_id = 3;
+      string message = 1;
+      string timestamp = 2;
+      float metric = 4;
+    };
+    ```
+
+1. Copy the `.proto` file to your ClickHouse server, into the directory set in the `proto_schema_path` option of your ClickHouse configuration.
+1. Reference that schema in the `clickhouse()` destination of your {{< product >}} configuration, like this:
+
+    ```sh
+    destination {
+      clickhouse(
+        database("default")
+        table("demo_table")
+        user("your-username")
+        password("your-password")
+        schema(
+          "user_id" UInt32 => $R_MSEC,
+          "message" String => "$MSG",
+          "timestamp" DateTime => "$R_UNIXTIME",
+          "metric" Float32 => 3.14
+        )
+        server-side-schema("my_proto_file_on_server:my_message_schema_name")
+      );
+    };
+    ```
+
+  <!-- FIXME where does the name of the schema come from? Filename? -->
```

Comment:
Yes, filename (with or without `.proto`) and the identifier after `message` (`MessageType`) in this case, combined together with a colon, for example: `my_schema:MessageType`.